lib/puppet/type/tidy.rb

```diff
@@ -310,6 +310,10 @@ def matches?(path)
 
   # Should we remove the specified file?
   def tidy?(path)
+    # ignore files that are already managed, since we can't tidy
+    # those files anyway
+    return false if catalog.resource(Puppet::Type.type(:file), path)
```

Comment:
I think this can be simplified to `catalog.resource(:file, path)`

Reply:
Fixed, thanks.